javascript/src/index.ts

```diff
@@ -72,6 +72,12 @@ export interface SvixOptions {
   serverUrl?: string;
 }
 
+const REGIONS = [
+  { region: "us", url: "https://api.us.svix.com" },
+  { region: "eu", url: "https://api.eu.svix.com" },
+  { region: "in", url: "https://api.in.svix.com" },
```

Comment:
Nitpick that you can ignore:
```
const REGIONS = {
    "us": { url: "https://api..." },
    ...
}
```
And then just `REGIONS[region]` below. It just feels very JS to me.